### ISSUE TITLE
SyntaxError when importing jiwer

### ISSUE THREAD

**Issue by vkuznetsovgn** [jitsi/jiwer#95]
Since Python 3.12 (Ubuntu 24) when trying to import jiwer it fails with the SyntaxError:

```
 /usr/local/lib/python3.12/site-packages/my_package/my_module.py:14: in <module>
     import jiwer
 /usr/local/lib/python3.12/site-packages/jiwer/__init__.py:1: in <module>
     from .measures import *
 /usr/local/lib/python3.12/site-packages/jiwer/measures.py:49: in <module>
     from jiwer import transforms as tr
 E     File "/usr/local/lib/python3.12/site-packages/jiwer/transforms.py", line 270
 E       """
 E       ^^^
 E   SyntaxError: invalid escape sequence '\w'
```

**Comment by nikvaessen** [comment]
Can you verify whether the current main branch fixed the issue? I think you can install it with pip via `pip install git+https://github.com/jitsi/jiwer.git@master`

**Comment by vkuznetsovgn** [comment]
Yes it's fixed!

**Comment by nikvaessen** [comment]
Should be fixed with version 3.0.5 on pipy.
